pose_evaluation/metrics/embedding_distance_metric.py

```diff
@@ -110,7 +89,7 @@ def _to_batch_tensor_on_device(self, data: TensorConvertableType, dtype=None) ->
             # https://stackoverflow.com/questions/55050717/converting-list-of-tensors-to-tensors-pytorch
             data = torch.stack(data)
 
-        return st_util._convert_to_batch_tensor(data).to(device=self.device, dtype=dtype)
```

Comment:
you can use `self.dtype` if you want to keep it consistent. my point was just that no need for `dtype=None` and then to check

Reply:
Ah right, that was my intention. And in fact that's important to do, and prevents some tests from failing.